### DISCUSSION TITLE
[Feature request] simple script for mass case

### DISCUSSION THREAD

**Discussion by slavonnet** [coqui-ai/TTS#3008] [General Q&A]
Hello. I have 10+ years of CC calls with transcribe. Please give universal solution for traming on that data. Need group by people or nerd full random? Need cutof dirty places or need use all of variants?:

In Russia  we have few thousands hours of Radio Fantasy Club. Dictor read fantasy books and DJ support him by full-on music in background.. One person 15+ yes do this few days in wvery werk.  And license is public (need only gets test part from books)

I think need give for everyone simple script for popular personal archive types)

**Comment by erogol** [comment]
Sorry but I couldnt understand what your feature request is

**Comment by slavonnet** [comment]
Hello. I have over 10 years of data  making calls to call centers using transcription. Can you please provide a universal solution for training based on this data in the form of a script and dataset format that is easy to use and produces results quickly?

Here are some questions t in head:

- Should the data be grouped or sorted by people/similarities, or is a random selection better?
- Is it better to use all available data or remove any dirty data?

Also In Russia, there are thousands of hours of audio stories read by the same announcer on the radio for over 10 years. The program is called "Model for Assembly" and consists of fantastic stories read by a dictor with calm background music provided by a DJ.

I believe it would be helpful to provide a simple script example for preparing data for popular data types such as call center calls, letters, and tasks. If it is possible to train the model privately and improve performance, many people will be interested in trying it. However, this is currently a difficult task.
